xrpl/model/transactions/payment.go

```diff
@@ -181,3 +183,71 @@ func (p *Payment) UnmarshalJSON(data []byte) error {
 
 	return nil
 }
+
+// ValidatePayment validates the Payment struct and make sure all the fields are correct.
+func ValidatePayment(tx FlatTransaction) {
+	ValidateBaseTransaction(tx)
+
+	// Check if the field Amount is set
+	if _, ok := tx["Amount"]; !ok {
+		panic("Missing field Amount")
```

Comment:
We cannot force a panic in the package. We should expose the error to the user, delegating the handling of the error